examples/seismic/skew_self_adjoint/example_iso.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from devito import configuration
+from examples.seismic import RickerSource
+from examples.seismic.skew_self_adjoint import *
+
+configuration['language'] = 'openmp'
```

Comment:
why do you need to force openmp? would it be too slow otherwise?

Reply:
sorry, just way I grew up around here hunting speed. Will remove. I understand you cant have that in scripts as the runner set the environment.

Reply:
done